braindecode/models/atcnet.py

```diff
@@ -207,23 +207,25 @@ def __init__(
         ])
 
         if self.concat:
-            self.max_norm_linears = nn.ModuleList([
+            self.final_layer = nn.ModuleList([
```

Comment:
@PierreGtch @brunaafl this one is quite tricky because it has different ways of processing depending on `self.concat` and so on. To really fulfill all the requirements we may need to create a custom `nn.Module` that would serve as the final layer`. I may think of sth tomorrow evening